editor/src/core/shared/project-file-types.ts

```diff
@@ -556,36 +556,36 @@ export interface TextFile {
   fileContents: TextFileContents
   lastSavedContents: TextFileContents | null // it is null when the file is saved
   lastParseSuccess: ParseSuccess | null
-  lastRevisedTime: number
+  versionNumber: number
```

Comment:
The similar type in `ClientModel.hs` needs to be updated too.

Reply:
Good catch! This was fixed in https://github.com/concrete-utopia/utopia/pull/3763/commits/c03047d5c6c9fbac89d53128e59eb63176606642